### PR TITLE
Update ec-helm-charts and give developer examples

### PR DESCRIPTION
**Pull request by marcelldls**

### CODE REVIEW

**Comment by marcelldls** [comment]
Requires https://github.com/epics-containers/ec-helm-charts/pull/15

File: template/services/{{ioc_group or beamline}}-ea-test-01/values.yaml [hunk 1]
```diff
@@ -2,3 +2,6 @@
 shared:
   ioc-instance:
     image: ghcr.io/epics-containers/ioc-adsimdetector-runtime:2024.4.1
+    # image: ghcr.io/epics-containers/ioc-adsimdetector-developer:2024.4.1  # Use a developer container
```
Comment (gilesknap) [inline template/services/{{ioc_group or beamline}}-ea-test-01/values.yaml:5]: I cant work out how to make 'suggested change' here.

But my suggested change is:

```

    # NOTE: the following are suggestions to help with debugging IOCs
    # 1. replace the runtime container with the developer version
    #    this adds tools and compilers to the container so you
    #    can experiment in the cluster to fix the issue
    #
    # image: ghcr.io/epics-containers/ioc-adsimdetector-developer:2024.4.1  # Use a developer container

    # 2. override the entry point to be a sleep infinity
    #    this means you can now use 'ec exec' to get a shell
    #    in the container and run /epics/ioc/start.sh.
    #    If it fails you can see the error and the pod
    #    will not terminate. In combination with 1. above you
    #    can experiment with changes until the issue
    #    is fixed.
    #
    # startCommand: sleep  # Override default entry point
    # startArgs: infinity  # Override default entry point
```

Comment (gilesknap) [reply template/services/{{ioc_group or beamline}}-ea-test-01/values.yaml:5]: Maybe you will argue that the detail should just go in the docs and the original examples are enough for people who understand what is going on. But the above text does need to go somewhere.

**Review by gilesknap — APPROVED** [review]
all good.

File: template/services/{{ioc_group or beamline}}-ea-test-01/values.yaml (hunk 1 above)
Comment (marcelldls) [reply template/services/{{ioc_group or beamline}}-ea-test-01/values.yaml:5]: Thanks Giles - I think this is a good suggestion for the test-ioc. Will add

**Comment by marcelldls** [comment]
@gilesknap Can we merge this?

**Comment by gilesknap** [comment]
sure thing. the merge button worked for me. 

Go ahead and release if you wish.

**Comment by marcelldls** [comment]
Are the release versions in sync with ec-helm-charts or is it coincidence?

**Comment by gilesknap** [comment]
I'd like to keep major.minor the same for them, see
https://epics-containers.github.io/main/explanations/changes.html#id1

**Comment by gilesknap** [comment]
especially as this is (almost) a breaking change that needs these two to sync

**Pull request merged**